_data/projects.yml

```diff
@@ -97,82 +98,60 @@ projects</a>, including <a href=\"https://www.vmware.com/\">VMware</a>,
         - *security
         - *internet_of_things
 
-
   - in-toto: &in-toto
       name: "in-toto"
       anchor: "in-toto"
       image: "img/projects/in-toto_diagram.png"
       status: *starting
       site: "https://in-toto.io"
-      description: "Do you know who has handled your software prior to its installation on your machine? Even if developers are careful to secure each step in their products' supply chain, there is little assurance about what happens in-between these steps. The in-toto system holistically enforces the integrity of a software supply chain by gathering and signing information about each step in the process. As such, in-toto provides accountability about how software is written, packaged and distributed...and by who."
-      products: "The in-toto software has already been integrated into several open source
-    projects. Most recently, Arch Linux announced that a patch to check for
-    invalid tags in git will be integrated into the next release of its <a href=\"https://git.archlinux.org/pacman.git/commit/?id=39319c1860d200a9b4a3cc2c6975e3cece502f2d\">pacman</a>
-    utility. We welcome you to download the in-toto
-    instructions, which includes a <a href=\"https://github.com/in-toto/in-toto/tree/develop/demo/\">demo version</a> of our software,
-    or to clone our <a href=\"https://github.com/in-toto/in-toto/\">repository</a> and follow the
-    directions to integrate in-toto into your software project!"
+      description: "Even if developers are careful to secure each step in
+  their products' supply chain, there is little assurance about what happens
+  in-between these steps. The in-toto system holistically enforces the integrity
+  of a software supply chain by gathering and signing information about each
+  step in the process. As such, in-toto provides accountability about how software
+  is written, packaged and distributed...and by who. We welcome you to download
+  the in-toto instructions, which includes a <a href=\"https://github.com/in-toto/in-toto/tree/develop/demo/\">demo version</a> of our software,
+  or to clone our <a href=\"https://github.com/in-toto/in-toto/\">repository</a>
+  and follow the directions to integrate in-toto into your software project!"
+      products: "The in-toto software has already been integrated into several
+  open source projects, including a new version of the Arch Linux
+  <a href=\"https://git.archlinux.org/pacman.git/commit/?id=39319c1860d200a9b4a3cc2c6975e3cece502f2d/">pacman</a>
+  utility that adds support for reproducible builds, and includes a security check
+  for tampered git tag metadata. The first version of the official
+  <a href=\"https://plugins.jenkins.io/in-toto/">in-toto Jenkins</a> was released
+  early in 2019. This provenance agent provides a secure and distributed way for
+  users to  track and sign link metadata for any step within a pipeline."
+In Progress: "Integrations are underway with repeatr and Debtain, and with the
```

Comment:
In some ways this is weaker. We have big integrations, so talking about these groups doesn't help so much.

Reply:
Did my best with these.